src/Marker.js

```diff
@@ -88,6 +99,10 @@ export default class Marker extends Component {
     }
   }
 
+  shouldComponentUpdate() {
```

Comment:
Please remove this for this PR. I want to implement `sCU` for components in another pull requests, thus making changes as small as possible.